public/app/features/dashboard-scene/sharing/ShareButton/ShareMenu.tsx

```diff
@@ -79,6 +77,20 @@ export default function ShareMenu({ dashboard, panel }: { dashboard: DashboardSc
       },
     });
 
+    customShareDrawerItems.forEach((d) => menuItems.push(d));
+
+    menuItems.push({
+      shareId: shareDashboardType.inviteUser,
+      testId: newShareButtonSelector.inviteUser,
+      icon: 'external-link-alt',
+      label: t('share-dashboard.menu.invite-user-title', 'Invite new member'),
+      renderCondition:
+        config.externalUserMngLinkUrl !== '' && contextSrv.hasPermission(AccessControlAction.OrgUsersAdd),
```

Comment:
```suggestion
        !!config.externalUserMngLinkUrl && contextSrv.hasPermission(AccessControlAction.OrgUsersAdd),
```
you check the truthiness. This also handles cases like null and undefined, making it more robust